storageClient/storageClient.go

```diff
@@ -213,6 +213,60 @@ func (c *StorageClient) EncryptMembershipKeyForGroupMember(ctx context.Context,
 
 }
 
+// GetGroupMembershipKey returns member information about the group
+func (c *StorageClient) GetGroupMembershipKey(ctx context.Context, params GetMembershipKeyRequest) (*ClientGroup, error) {
+	var result *ClientGroup
+	path := c.Host + storageServiceBasePath + "/groups/" + params.GroupID.String() + "/membership_key"
+	req, err := e3dbClients.CreateRequest("GET", path, params)
+	if err != nil {
+		return result, err
+	}
+	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, req, c.SigningKeys, c.ClientID, &result)
+	return result, err
+}
+
+// CreateGroupAccessKey
+func (c *StorageClient) CreateGroupAccessKey(ctx context.Context, params GroupAccessKeyRequest) (e3dbClients.SymmetricKey, string, error) {
+	var wrappedAccessKey string
+	var accessKey e3dbClients.SymmetricKey
+	// Decrypt Access Key
+	rawEncryptionKey, err := e3dbClients.DecodeSymmetricKey(c.EncryptionKeys.Private.Material)
+	if err != nil {
+		return accessKey, wrappedAccessKey, err
+	}
+	accessKey, err = e3dbClients.DecryptEAK(params.EncryptedAccessKey, c.EncryptionKeys.Public.Material, rawEncryptionKey)
+	if err != nil {
+		return accessKey, wrappedAccessKey, err
+	}
+	// Encrypt Access Key with Group's Public Key
+	groupPubKey := params.PublicKey
+	encryptionKeys := e3dbClients.EncryptionKeys{
+		Public: e3dbClients.Key{
+			Type:     e3dbClients.DefaultEncryptionKeyType,
+			Material: groupPubKey,
+		},
+		Private: c.EncryptionKeys.Private,
+	}
+	eak, eakN, err := e3dbClients.BoxEncryptToBase64(accessKey[:], encryptionKeys)
```

Comment:
You can do better than me. Lines 250 - 254 should be extracted to a re-usable function that can be used both here and from the original function CreateSharingAccessKey this is based off of.